docs/source/en/model_summary.md

```diff
@@ -16,7 +16,7 @@ rendered properly in your Markdown viewer.
 
 # The Transformer model family
 
-Since its introduction in 2017, the [original Transformer](https://arxiv.org/abs/1706.03762) model has inspired many new and exciting models that extend beyond natural language processing (NLP) tasks. There are models for [predicting the folded structure of proteins](https://huggingface.co/blog/deep-learning-with-proteins), [training a cheetah to run](https://huggingface.co/blog/train-decision-transformers), and [time series forecasting](https://huggingface.co/blog/time-series-transformers). With so many Transformer variants available, it can be easy to miss the bigger picture. What all these models have in common is they're based on the original Transformer architecture. Some models only use the encoder or decoder, while others use both. This provides a useful taxonomy to categorize and examine the high-level differences within models in the Transformer family, and it'll help you understand Transformers you haven't encountered before.
+Since its introduction in 2017, the [original Transformer](https://arxiv.org/abs/1706.03762) model has inspired many new and exciting models that extend beyond natural language processing (NLP) tasks. There are models for [predicting the folded structure of proteins](https://huggingface.co/blog/deep-learning-with-proteins), [training a cheetah to run](https://huggingface.co/blog/train-decision-transformers), and [time series forecasting](https://huggingface.co/blog/time-series-transformers). With so many Transformer variants available, it can be easy to miss the bigger picture. What all these models have in common is they're based on the original Transformer architecture. Some models only use the encoder or decoder, while others use both. This provides a useful taxonomy to categorize and examine the high-level differences within models in the Transformer family, and it'll help you understand Transformers you haven't encountered before. For a gentle introduction see the [Annotated Transformer](http://nlp.seas.harvard.edu/2018/04/03/attention.html).
```

Comment:
### **=== ✅ Suggestion Implemented ===**

I actually do like that a lot more, thanks @stevhliu !

Updated my PR in two commits to match your suggestion.
_(Minor fyi: Couldn't commit it directly because the suggestion parenthetical had an extra `)`)_

- [3d0ec4d](https://github.com/huggingface/transformers/pull/29702/commits/3d0ec4db7ba933a780fa6b581e376cec640e8e89)
   - matches your suggestion's **wording** + **placement of link**
   - but typo: commit accidentally removed "**has**" when I pasted the suggestion
- [9378f4d](https://github.com/huggingface/transformers/pull/29702/commits/9378f4d42623184947893b7ac37a6abb30f8ebd3)
   - restores "**has**" after the parenthetical

<br>

### **=== VS Code: Diff Screenshots ===**

GitHub struggles with single-line diffs, so here's my local diff in VS Code.
Just OCD, but feel free to verify for yourself too :)

<details>
<summary>diff for 3d0ec4d -- note the oops, missing "has"</summary>

![image](https://github.com/huggingface/transformers/assets/65138515/698d5581-ebc6-4a42-99fd-f0f8cf238135)

</details>
<details>
<summary>diff for 9378f4d -- "has" restored</summary>

![image](https://github.com/huggingface/transformers/assets/65138515/e65aaa50-1f9d-4dde-860d-a8d8a1cc1f3f)

</details>

And finally: current PR's overall diff with `main` from last week

![image](https://github.com/huggingface/transformers/assets/65138515/f271bd74-7339-458d-8dd5-dc1e3db8c1d5)

Reply:
If there are no more change requests, feel free to merge it whenever.
And if there are any typos I missed, feel free to directly update the PR.

**Sidenote:**
I do like the annotated link much much better next to the original Transformer link, but I was hesitant to add too much noise to the intro sentence as a newcomer contributer. So thanks for that suggestion. 👍

**PS:**
I really appreciate the responsiveness and culture in this project, thanks for the help and the positive experience. :)